docs/upgrading-to-70.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[upgrading-to-70]]
+=== Upgrading to APM Server v7.0
+
+Before upgrading to APM Server v7.0,
+there are some {apm-overview-ref-v}/breaking-7.0.0-beta1.html[breaking changes]
```

Comment:
out of curiosity, this link (breaking-7.0.0-beat1) is permanent or is intended to be changes to breaking-7.0 eventually?

Reply:
I was planning on changing all of these (beta-1, rc1, etc.) to 7.0.0 when it releases.